src/app/courses/components/courses-list/courses-list.component.spec.ts

```diff
@@ -3,8 +3,9 @@ import { ComponentFixture, TestBed } from '@angular/core/testing';
 import { CoursesListComponent } from './courses-list.component';
 
 describe('CoursesListComponent', () => {
-  let component: CoursesListComponent;
+  let component: CoursesListComponent = new CoursesListComponent();
   let fixture: ComponentFixture<CoursesListComponent>;
+  console.log = jasmine.createSpy('log');
 
```

Comment:
That's cool. You should use the same technique to spy on emit event in course-card component.